config/locales/en.yml

```
@@ -180,6 +180,7 @@ en:
       call_list: Your call list
       completed_calls: Your completed calls
       search_results: Search results
+      unconfirmed_support: Unconfirmed Support
```

sorry to be a bear about this but 'unconfirmed support' is slightly intense verbiage. Let's bump slightly:
```suggestion
      unconfirmed_support: Unconfirmed Practical Support
```